### ISSUE TITLE
Ont App Fails on Keycloak security

### ISSUE THREAD

**Issue by valhuber** [ApiLogicServer/ApiLogicServer-src#66]
Stacktrace:

```
File "/Users/val/dev/ApiLogicServer/samples/nw_sample/security/authentication_provider/keycloak/auth_provider.py", line 128, in get_user_from_jwt
    rtn_user.name = jwt_data["preferred_username"]
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
```

The creation sequence:
1. `als start`, which creates default apps
2. open `nw_sample` - verify runs with sra
3. set for `als add-auth --provider-type=keycloak`
4. Use Run configs to prepare/start Ont, and restart als
5. Try to login to Ont - fails with stack trace above
6.

**Comment by tylerm007** [comment]
als add-auth --provider-type=keycloak --db-url=localhost

worked for me testing ontimize

**Comment by tylerm007** [comment]
One a side note - Our design of how we pass the auth_token will not work at scale - since it was designed as a global - the 2nd login may break.  Needs Flask redesign and multi-user test setup.

**Comment by valhuber** [comment]
Is it the case that I must ont build after changing security?

**Comment by valhuber** [comment]
Fixed 11.00.04
